testsuite/tests/typing-layouts-or-null/containers.ml

```diff
@@ -131,7 +131,7 @@ Line 1, characters 28-32:
 Error: This expression has type "'a or_null"
        but an expression was expected of type "('b : value)"
        The kind of 'a or_null is value_or_null
-         because it is the primitive value_or_null type or_null.
+         because it is the primitive immediate_or_null type or_null.
```

Comment:
this seems wrong now? as in, either the first line should change to say `immediate_or_null` or the second line should say `value_or_null` still.

Reply:
Hm, yeah. This seems like a with-kinds issue. You should be able to do the same with lists, `value` and `immutable_data`.

Reply:
can you leave a CR?

Reply:
done.